docs/spec_v7.md

```diff
@@ -208,7 +208,7 @@ corresponding HTTP Status Code for the response.
 | already_exists      | 409 | An attempt to create an entity failed because one already exists.
 | permission_denied   | 403 | The caller does not have permission to execute the specified operation. It must not be used if the caller cannot be identified (use "unauthenticated" instead).
 | unauthenticated     | 401 | The request does not have valid authentication credentials for the operation.
-| resource_exhausted  | 403 | Some resource has been exhausted, perhaps a per-user quota, or perhaps the entire file system is out of space.
+| resource_exhausted  | 429 | Some resource has been exhausted, perhaps a per-user quota, or perhaps the entire file system is out of space.
```

Comment:
Is it worth mentioning rate-limiting here?

Reply:
What about adding "exhausted or rate-limited"?
```
Some resource has been exhausted or rate-limited, perhaps a per-user quota, or perhaps the entire file system is out of space
```